buildtest/utils/command.py

```diff
@@ -1,7 +1,69 @@
+from buildtest.utils.file import read_file
+import os
 import locale
 import subprocess
 import shlex
 import shutil
+import tempfile
+
+
+class Capturing:
+    """capture output from stdout and stderr into capture object.
+       This is based off of github.com/vsoch/gridtest but modified
+       to write files. The stderr and stdout are set to temporary files at
+       the init of the capture, and then they are closed when we exit. This
+       means expected usage looks like:
+
+       with Capturing() as capture:
+           process = subprocess.POpen(...)
```

Comment:
typo **POpen** --> **Popen**

Reply:
It may be nice to add full example in docstring of how to use ``subprocess.Popen`` with Capture class in docstring as pose to **...**

Reply:
Sure, feel free to add this after if you like!

Reply:
Popen typo fixed!

Reply:
Thanks